data/curated/pokemon/intro.md

```diff
@@ -0,0 +1,19 @@
+<!-- 
+1. Describe the dataset. See previous weeks for the general format of the
+DESCRIPTION. The description is the part of the readme.md file above "The Data";
+everything else will be filled in from the other md files in this directory +
+automatic scripts. We usually include brief introduction along the lines of
+"This week we're exploring DATASET" or "The dataset this week comes from 
+SOURCE", then a quote starting with ">", then a few questions participants might
+seek to answer using the data. 
+2. Delete this comment block.
+-->
+DESCRIPTION
+
+This week we are exploring Pokemon! This dataset is sourced from a CRAN library which provides Pokemon information in both English and Brazilian Portuguese. (also on GitHub: https://github.com/williamorim/pokemon/)
+
+The dataset contains 949 rows and 22 columns, including information such as the Pokemon’s name, ID, height, weight, stats, type, and more.
+
+> A dataset of Pokemon information
+
+QUESTION?
```

Comment:
```suggestion
```
This is intended as a prompt to you, to add one or more questions to the end of the introduction.